client/src/index.js

```diff
@@ -3,9 +3,17 @@ import ReactDOM from 'react-dom';
 import 'bootstrap/dist/css/bootstrap.css';
 import App from './App';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+// Gives preloader additional 2 secs to perform
+// before mounting React app
+setTimeout(
+  () => {
+    ReactDOM.render(
+      <React.StrictMode>
+        <App />
+      </React.StrictMode>,
+      document.getElementById('root')
+    );
+  },
+  2000
```

Comment:
how about 1500 or 1000?

Reply:
Don't know about this, I'm up for suggestions!

Reply:
suggestions?

Reply:
2 seconds feels too long to me. 1500 would be better.

Reply:
done!